test/Test/Database/LSMTree/Internal/Entry.hs

```diff
@@ -31,188 +27,71 @@ tests = adjustOption (\_ -> QuickCheckTests 10000) $
 
       testClassLaws "Regular Entry" $
         semigroupLaws (Proxy @(Regular (Entry (Sum Int) String)))
-    , testClassLaws "Regular Unified.Update" $
-        semigroupLaws (Proxy @(Regular (Unified.Update (Sum Int) String)))
-    , testClassLaws "Regular Normal.Update" $
-        -- Note that we are using Unlawful here because we do not combine values
-        -- monoidally.
-        semigroupLaws (Proxy @(Regular (Normal.Update (Unlawful Int) String)))
-    , testClassLaws "Regular (Monoidal.Update)" $
-        semigroupLaws (Proxy @(Regular (Monoidal.Update (Sum Int))))
+    , testClassLaws "Regular Update" $
+        semigroupLaws (Proxy @(Regular (Update.Update (Sum Int) String)))
 
     , testClassLaws "Union Entry" $
         semigroupLaws (Proxy @(Union (Entry (Sum Int) String)))
-    , testClassLaws "Union Unified.Update" $
-        semigroupLaws (Proxy @(Union (Unified.Update (Sum Int) String)))
-    , testClassLaws "Union Normal.Update" $
-        semigroupLaws (Proxy @(Union (Normal.Update (Sum Int) String)))
-    , testClassLaws "Union Monoidal.Update" $
-        semigroupLaws (Proxy @(Union (Monoidal.Update (Sum Int))))
+    , testClassLaws "Union Update" $
+        semigroupLaws (Proxy @(Union (Update.Update (Sum Int) String)))
 
     -- * Semantics
 
-    , testProperty "prop_regularSemantics_unified" $
-        prop_regularSemantics_unified @(Sum Int) @String
-    , testProperty "prop_regularSemantics_normal" $
-        -- Note that we are using Unlawful here because we do not combine values
-        -- monoidally.
-        prop_regularSemantics_normal @(Unlawful Int) @String
-    , testProperty "prop_regularSemantics_monoidal" $
-        prop_regularSemantics_monoidal @(Sum Int)
-
-    , testProperty "prop_unionSemantics_unified" $
-        prop_unionSemantics_unified @(Sum Int) @String
-    , testProperty "prop_unionSemantics_normal" $
-        prop_unionSemantics_normal @(Sum Int) @String
-    , testProperty "prop_unionSemantics_monoidal" $
-        prop_unionSemantics_monoidal @(Sum Int)
+    , testProperty "prop_regularSemantics" $
+        prop_regularSemantics @(Sum Int) @String
+
+    , testProperty "prop_unionSemantics" $
+        prop_unionSemantics @(Sum Int) @String
     ]
 
--- TODO: it would be nice to write down how the semantic tests below relate to
+-- TODO: it would be nice to write down how the semantic test below relates to
 -- the semantics of operations on the public API.
```

Comment:
It should still be "tests" plural

Reply:
Fixed in 15cb6bc4163c74ee70be251e20f70a38f71b35d7.